Formula/php@8.0.rb

```diff
@@ -23,7 +23,7 @@ class PhpAT80 < Formula
 
   keg_only :versioned_formula
 
-  deprecate! date: "2023-11-26", because: :versioned_formula
+  deprecate! date: "2022-11-26", because: :versioned_formula
```

Comment:
The page you linked says security support until 2023?

Reply:
Yes, so it's deprecated from 2022 and EOL in 2023.

Reply:
Ok, that makes sense. I was comparing dates with the other formulae so it confused me a little.

Reply:
How do you feel about adding a commented disable for that? Ideally we can define both in one go, but unfortunately it doesn't work that way.

Reply:
A commented disable makes sense to me.